### DISCUSSION TITLE
index.d.ts from [@types/ramda] export for pick function has typescript errors

### DISCUSSION THREAD

**Discussion by JeffreyStevens** [DefinitelyTyped/DefinitelyTyped#67446] [Issues with a @types package] [answered]
My dependencies include the following:
  ```
"@types/ramda": "^0.29.9",
"ramda": "^0.29.1",
"typescript": "^4.8.4",
```

when my Webstorm IDE scans for TypeScript project errors the exported `pick` function shows 24 errors:

![Screenshot 2023-11-19 at 5 34 08 AM](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/34170674/e17800b4-ef04-443e-91bb-f0687fa5787f)

[a comment by an automated account is not shown]

**Comment by Harris-Miller** [comment, marked as answer]
@JeffreyStevens min typescript version is 5: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ramda/package.json#L21

**Reply by Harris-Miller** [reply]
I should mention I'm working on a `<5` typescript version as described here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-very-new-typescript-versions

**Comment by JeffreyStevens** [comment]
That works, thank you!
